src/site/content/en/notifications/codelab-notifications-get-started/index.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: codelab
+title: Get started with the Notifications API
+description: |
+  In this codelab, learn how to request user permission and send notifications.
+date: 2019-10-15
+# Add an updated date to your post if you edit in the future.
+# updated: 2019-06-27
+glitch: codelab-notifications-get-started
+# Specify which file the glitch should start on.
+glitchPath: public/index.js
+related_post: use-push-notifications-to-engage-users
+---
+
+In this codelab, you'll use basic features of the
+[Notifications API](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API) to:
+
+* Request permission to send notifications
+* Send notifications
+* Experiment with notification options 
+
+## Remix the sample app and view it in a new tab
+
+Notifications are automatically blocked from the embedded Glitch app, so you won't be able to preview the app on this page. Instead, here's what to do:
+
+{% Instruction 'remix', 'ol' %}
+{% Instruction 'preview', 'ol' %}
+
+The Glitch should open in a new Chrome tab:
+
+<img class="w-screenshot" src="./live-app-new-tab.png" alt="Screenshot showing the remixed live app in a new tab">
+
+As you work through this codelab, make changes to the code in the embedded Glitch on this page. Refresh the new tab with your live app to see the changes.
+
+## Get familiar with the starting app and its code
+
+Start by checking out the live app in the new Chrome tab:
```

Comment:
Despite the title, my inefficient brain didn't immediately catch that this section is divided into a checking-out-the-app part and a checking-out-the-code part. I restructured a bit to hopefully make that more apparent for other inattentive readers like me and to reduce some of the list nesting. (Since there's so little horizontal real estate in the codelab format, I think we should try to avoid multi-level lists whenever possible to keep lines at an easily readable length.)

Reply:
Thx, I like the restructure. 

Per inline comment I don't like to use "Check out this thing" synonymously with "Look at/do this thing" because "check out" could refer to checking out a git repo